src/api/app/views/webui2/webui/project/new_package.html.haml

```diff
@@ -0,0 +1,7 @@
+- @pagetitle = "Create package for #{@project}"
+
+.card.mb-3
+  = render partial: 'tabs', locals: { project: @project }
+  .card-body
+    %h3= @pagetitle
+    = render partial: 'create_package_form', locals: { project: @project }
```

Comment:
The cancel button doesn't have any sense on this page. Could you remove it?, please :smile_cat:

Reply:
True! Please have another look. I updated the screenshots in the PR's description.